packages/web/src/app/doc-pages/brand/illustrations-doc/illustrations-data.ts

```diff
@@ -0,0 +1,17 @@
+import changelogJson from '@elvia/elvis-illustrations/CHANGELOG.json';
+
+import ComponentData from '../../components/component-data.interface';
+
+export type IllustrationColor = 'grey' | 'purple' | 'green' | 'blue' | 'orange';
+
+export const illustrationsData: ComponentData<{ color: IllustrationColor }> = {
```

Comment:
This requires us to mirror the public API for our component. However we don't need to over-engineer it yet imo 🙂

Reply:
Yeah. thats what I was thinking. And since the component is purely a custom element, there isn't really much point in exposing an interface for it, at least yet